plugins/tail.yml

```diff
@@ -0,0 +1,28 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: tail
+spec:
+  version: "v0.9.1"
+  platforms:
+  - uri: https://github.com/boz/kail/releases/download/v0.9.1/kail_0.9.1_darwin_amd64.tar.gz
+    sha256: 3824de171a73c773649401f2bdccbd138f0022333c69e39e4f194e8ebd08290b
+    bin: kail
+    files:
+    - from: "kail"
+      to: "."
+    selector:
+      matchExpressions:
```

Comment:
I think you should also specify the supported architecture of the binary. This is better done with the `matchLabels` shorthand:
```yaml
selector:
  matchLabels:
    os: darwin
    arch: amd64
```

Reply:
yeah if this block is just for darwin, `matchLabels:` is a better syntax.